tfhe/src/high_level_api/booleans/compact.rs

```diff
@@ -148,10 +174,11 @@ impl<'a> FheTryEncrypt<&'a [bool], CompactPublicKey> for CompactFheBoolList {
     ///
     /// See [CompactFheBoolList] example.
     fn try_encrypt(values: &'a [bool], key: &CompactPublicKey) -> Result<Self, Self::Error> {
-        let ciphertext = key
```

Comment:
we don't have boolean compact lists at the integer level, do you think it's something we would want to have ?

I'm perfectly fine with that wrapper at the HL level

Reply:
We potentially could, also I don't think there is a SignedCompactList at the integer level

Reply:
ok I'll create a follow up issue for this then

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/516